src/v3/src/transformer/selectAuthenticator/utils.test.ts

```diff
@@ -263,8 +266,13 @@ describe('Select Authenticator Utility Tests', () => {
         expect(currentOption?.label).toBe(option.label);
         expect(currentOption?.options.ctaLabel)
           .toBe('oie.verify.authenticator.button.text');
-        expect(currentOption?.options.description)
-          .toBe(option.relatesTo?.key === AUTHENTICATOR_KEY.PHONE ? mockPhoneNumber : undefined);
+        let expeectedDescription;
```

Comment:
nit: ideally avoid the `let` for expected value and instead we can have different assertions in each block, or we can have separate tests